### ISSUE TITLE
no example code?

### ISSUE THREAD

**Issue by j4cobgarby** [Parou/Cocoa-Biscuit-Cake-with-Marshmallow-Cheese-Creme#5]
very hard to follow with no example code or screenshots

**Comment by Parou** [comment]
Thank you for your suggestion.
Here is one of the code examples you requested: https://github.com/Parou/Cocoa-Biscuit-Cake-with-Marshmallow-Cheese-Creme/tree/master/php

~ Alice

**Comment by Parou** [comment]
SanitizedOutput also posted some pictures of his try at https://devrant.com/rants/1402213/guys-i-finally-did-something-useful-on-github-look-at-my-unique-project-https-gi
